docs/Style-Guide.md

```diff
@@ -128,7 +128,24 @@ Remember: if you can use a computed property based on the vuex store, it means y
 
 Virtually everything in this project is [going to be converted](https://github.com/okTurtles/group-income-simple/issues/295) to SBP ("selector-based programming", see `shared/sbp.js`).
 
-Details about SBP will be written in a blog post soon. In the meantime, you are encouraged to adopt this paradigm wherever possible for your own code.
+In SBP everything works based on selectors. A selector is a string composed by two parts: a _domain_ and an _action_. For example, in `'okTurtles.data/set'`, the domain is `okTurtles.data` and the action is `/set`.
+The first argument of `sbp` is always a selector you created before and the rest of the arguments are arbitrary data to be later processed by the selector.
```

Comment:
"you created" => "you registered"
"arbitrary data" => "parameters"
"by the selector" => "by the function assigned to the selector"